service_discovery_manager/Component/mmBASE/BaseAPI/ifaddrs.cc

```diff
@@ -0,0 +1,221 @@
+/*
+ *  Copyright 2012 The WebRTC Project Authors. All rights reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree. An additional intellectual property rights grant can be found
+ *  in the file PATENTS.  All contributing project authors may
+ *  be found in the AUTHORS file in the root of the source tree.
+ */
```

Comment:
This license is not full license, LICENSE file is not exist in the root of SDM. Is it OK?

Reply:
added on latest patch-set